tests/ElementFinderFactoryTest.php

```diff
@@ -70,8 +71,7 @@ public function getDifferentCharsetStylesDataProvider(): array
 
     /**
      * @dataProvider getDifferentCharsetStylesDataProvider
-     * @param string $html
-     * @param string $bodyText
+     * @throws Exception
```

Comment:
this method does not throw an exception.

Reply:
An exception can be thrown in methods that are used in this method. The methods calls structure is similar to the one in this issue: https://youtrack.jetbrains.com/issue/WI-941

Reply:
Ok, let it be so. But what is the reason to add this tag here?

Reply:
Because an exception is possible in this method. And my linter also asks to indicate this.

Reply:
Yeah, but you should not follow all linter rules because someone told it to you. 
My linter is not complaining) 

My question was the following: why you think that we need to indicate expectation in this test method?

Reply:
I'll just delete it.